### ISSUE TITLE
Accessibility issue: Landmarks should have a unique role or role/label/title (i.e. accessible name) combination on /services/

### ISSUE THREAD

**Issue by anabil25** [anabil25/hi-state-services#15]
## What
An accessibility scan flagged the element `<nav class="utility-nav">
      <a href="/about/#contact">Contact</a>
      <a href="" class="tty-link"></a>
      <a href="/forms/">Forms</a>
    </nav>` because landmarks should have a unique role or role/label/title (i.e. accessible name) combination. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/landmark-unique?application=playwright.

[View screenshot](https://github.com/anabil25/hi-state-services/blob/gh-cache/.screenshots/4e6c4a9d-0323-4502-be83-d3c7c1283703.png)

To fix this, ensure landmarks are unique.

Specifically:

Fix any of the following:
-   The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
